### PR TITLE
prep 1.2.0 release

### PR DESCRIPTION
**Pull request by gthao313**
**Issue number:**
#479 

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

**Review by stmcginnis — APPROVED** [review]
🎉

**Review by etungsten — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,34 @@
...
+
+* Refactored deployment and yaml generation to use Helm templating ([#126])
+* Improved time window to become cron expression based scheduler ([#343]), ([#428])
+
+Fixed
+
+* Removed unnecessary dep on older "time" crate from chrono ([#415])
+* Fixed metrics not working for ipv6 clusters ([#406])
+* Makefile: refactors brupop-image target ([#418])
+* Cargo: use env vars when calling cargo ([#461])
```
Comment (etungsten) [inline CHANGELOG.md]: #461 has not been closed. Is that intentional?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,34 @@
+# 1.2.0
+
+## General
+
+Added
+
+* Refactored deployment and yaml generation to use Helm templating ([#126])
+* Improved time window to become cron expression based scheduler ([#343]), ([#428])
```
Comment (etungsten) [inline CHANGELOG.md:8]: #428 has not been closed.

Comment (gthao313) [reply CHANGELOG.md:8]: The root cause for this issue was we hadn't release new versions which contains the new features on dev branch, so users reported the errors about mismatch containers. @stmcginnis had a related PR which is about improvement. Once we have new version out, this issue will be closed.

File: CHANGELOG.md (hunk 1 above)
Comment (gthao313) [reply CHANGELOG.md]: We currently have a stopgap solution to fix the issue, but eventually we will find another solution to replace it.

**Review by etungsten — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (etungsten) [reply CHANGELOG.md]: Is the stop gap solution part of this release? Can you link the PR instead of the open issue?

Comment (gthao313) [reply CHANGELOG.md]: Yeah, otherwise. the helm chart would not work.  Sure, I'll replace the link to the PR.

**Review by etungsten — APPROVED** [review]

**Pull request merged**